plugins/available/blesh.plugin.bash

```diff
@@ -0,0 +1,9 @@
+# shellcheck shell=bash
+cite about-plugin
+about-plugin 'load ble.sh, the Bash line editor!'
+
+if [[ -f "$HOME/.local/share/blesh/ble.sh" ]]; then
+	source "$HOME/.local/share/blesh/ble.sh"
```

Comment:
I think @gaelicWizard is wanting to make us XDG compliant, so this is probably a good beginning step in that direction.

Reply:
[Resolve conversation]

I'm fine with closing the conversation. Since I'm neither the author of this PR nor a Collaborator of the repository, I cannot close the conversation. Could someone press the button?